browse/controllers/list_page/paging.py

```diff
@@ -0,0 +1,65 @@
+"""Paginiation links for listing pages."""
+
+
+from typing import Any, Dict, List, Union
+import math
+from flask import url_for
+
+
+def paging(count: int, skipn: int, shown: int, context: str, subcontext: str) \
+    -> List[Dict[str, Union[str, int]]]:
+    """Get paging links."""
+    B = 3  # num of buffer pages on each side of current
+    L = math.floor(count-1 / (skipn+1))+1  # total number of pages
+    S = 2 * B + 2 * 2 + 1  # number of total links in the pages sections:
+    #  2*buffer + 2*(first number + dots) + current
+
+    def page_dict(n: int, nolink: bool = False)->Dict[str, Union[str, int]]:
+        txt = str(n+1)+'-'+str(min(count, n+shown))
```

Comment:
[PEP008 recommends](https://www.python.org/dev/peps/pep-0008/#other-recommendations) whitespace around operators. Also, consider an F-string for readability.
```suggestion
        txt = f'{n + 1}-{min(count, n + shown)}'
```